src/omnistap/Utils/ogFileUtils/$write.omh

```diff
@@ -33,7 +33,11 @@ If lbOK=kFalse
 	Quit method kFalse
 End If
 
-Do loFile.$writecharacter(pnEncoding,pcText,pbAppend) Returns lbOK
+Do uniconv(kUniTypeUTF32,pcText,pnEncoding,Bin,kFalse,Error)

```

Comment:
Please rename these new variables to `lbnBin` and `lcError` to keep with the existing naming conventions.